src/guides/v2.3/get-started/authentication/gs-authentication.md

```diff
@@ -165,6 +159,43 @@ Each type of client has a preferred authentication method. To authenticate, use
    </tr>
 </table>
 
+The following sections describe tips about which type of authentication method you should use depending on your use case.
+
+### Token based (Bearer Authentication)
+
+This method is a good choice for authenticating customers and Admin users in third-party applications that need to make authorized API calls to the Magento store.
+
+*  **Customer Token**—Use this token in applications to authorize specific customer and query data related to that customer (for example, customer details, cart, and orders).
+*  **Admin Token**—Use this token in applications to authorize an Admin user and access Admin-related APIs.
+
+[Request a token]({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#request-token) and then (include it in future requests)({{ page.baseurl }}/get-started/authentication/gs-authentication-token.html#web-api-access).
+
+{:.bs-callout-info}
+You should use this type of authentication mechanism over HTTPS:
```

Comment:
looks like colon should be replaced with dot

Reply:
You're right @xmav. Thanks.